blog/zh/blog/2023/09/13/biweekly-report.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "社区双周报 (8.28 - 9.10)"
+keywords: 
+- Apache APISIX
+- API 网关
+- 社区周报
+- 贡献者
+description: “独行者速，众行者远”。Apache APISIX 社区周报希望可以帮助社区小伙伴们更好地掌握 Apache APISIX 社区的进展，方便大家参与到 Apache APISIX 社区中来。
+tags: [Community]
+image: https://static.apiseven.com/uploads/2023/09/11/Co3naagN_%E7%A4%BE%E5%8C%BA%E5%8F%8C%E5%91%A8%E6%8A%A5-%E5%85%AC%E4%BC%97%E5%8F%B7%E5%A4%B4%E5%9B%BE-%E4%B8%AD%E6%96%87-0911.png
+---
+
+> 我们近期对 Apache APISIX 和 APISIX Ingress Controller 的部分功能进行了修复或改善，包括为 `wasm` 插件提供原生 json 数据结构入参支持、为长亭 `WAF` 插件增加 UNIX sock 监听支持、以及支持在 APISIX Ingress 中移除 APISIX 自身需要的 etcd，减少架构复杂度。详情请阅读本期双周报。
+
+<!--truncate-->
+
+## 导语
+
+Apache APISIX 从开源第一天就以社区方式成长，迅速成为全世界最活跃的开源 API 网关项目。这些成就，离不开社区小伙伴们的共同奋斗。
+
+从 2023.8.28 - 2023.9.10，有 16 位开发者为 Apache APISIX 提交了 47 个 commit。感谢这些小伙伴为 Apache APISIX 添砖加瓦，是你们的无私付出，让 Apache APISIX 项目变得更好！
+
+我们近期对部分功能进行了修复或改善，更新内容汇总如下：
+
+1. 为 `wasm` 插件提供原生 json 数据结构入参支持
+
+2. 为长亭 `WAF` 插件增加 UNIX sock 监听支持
+
+3. 支持在 APISIX Ingress 中移除 APISIX 自身需要的 etcd，减少架构复杂度（实验特性）
+
+“独行者速，众行者远”。Apache APISIX 社区周报希望可以帮助社区小伙伴们更好地掌握 Apache APISIX 社区的进展，方便大家参与到 Apache APISIX 社区中来。
+
+## 贡献者统计
+
+![贡献者名单](https://static.apiseven.com/uploads/2023/09/11/fQoyADFJ_contributors%20list%201-0911.png)
+
+![新晋贡献者](https://static.apiseven.com/uploads/2023/09/11/UVL4ZB8S_%E6%96%B0%E6%99%8B%E8%B4%A1%E7%8C%AE%E8%80%85%E6%B5%B7%E6%8A%A5-0911.png)
```

Comment:
wrong picture and bad link
New contributors: SuperBo,zhangyuthink,JYan00,zclaiqcc,utkar-sh-ukla